setupext/build_ext.py

```diff
@@ -311,10 +311,16 @@ def build_java_ext(self, ext):
             cmd1 = shlex.split('%s -cp "%s" -d "%s" -g:none -source %s -target %s -encoding UTF-8' %
                                (javac, classpath, build_dir, target_version, target_version))
             cmd1.extend(ext.sources)
+            self.announce("  %s" % " ".join(cmd1), level=distutils.log.INFO)
+            subprocess.check_call(cmd1)
 
-            os.makedirs("build/classes", exist_ok=True)
+            cmd1 = shlex.split('%s -cp "%s" -d "%s" -g:none -source %s -target %s -encoding UTF-8' %
+                               (javac, build_dir, os.path.join(build_dir,"META-INF","versions","0"), target_version, target_version))
+            cmd1.extend(glob.glob(os.path.join("native","java0","**","*.java"), recursive=True))
+            os.makedirs(os.path.join(build_dir,"META-INF","versions","0"), exist_ok=True)
```

Comment:
Inconsistent use of whitespace between function parameters.

Just pretend I'm an annoying bot 😛.

Reply:
My autopep8 hook appears to have broken.   It is supposed to reformat on each commit.